.github/workflows/tpu-tests.yml

```diff
@@ -26,7 +27,10 @@ jobs:
   test-on-tpus:
     runs-on: ubuntu-22.04
     # run only when the PR title contains '[TPU]' or is a merge to master
-    if: ${{ startsWith(github.event_name, 'pull_request') && contains(github.event.pull_request.title, '[TPU]') || (github.event_name == 'push' && github.ref == 'refs/heads/master') }}
+    if: |
+      (github.event_name == 'push' && github.ref == 'refs/heads/master') ||
+      (startsWith(github.event_name, 'pull_request') && ( contains(github.event.pull_request.title, 'TPU'))
```

Comment:
```suggestion
      (startsWith(github.event_name, 'pull_request') && ( contains(github.event.pull_request.title, '[TPU]'))
```

Reply:
Why brackets? There casses when it is not exclusive for TPU but it is touched...

Reply:
well I think having some FP is not a bug deal, then have missed cases

Reply:
Brackets is probably easiest. Only us the maintainers need to know this, since we can update the titles for contributors if needed.